app/controllers/projects/menus_controller.rb

```diff
@@ -90,8 +95,7 @@ def query_item_selected?(id)
     end
 
     def no_query_props_or_all?
```

Comment:
I think the method name `no_query_props_or_all?` no longer applies. How about just `no_query_props?`


```suggestion
    def no_query_props?
```